kong/plugins/openwhisk/handler.lua

```diff
@@ -106,13 +101,14 @@ function OpenWhisk:access(config)
 
   -- only allow POST
   if var.request_method ~= "POST" then
-    return responses.send_HTTP_METHOD_NOT_ALLOWED()
+    return kong.response.exit(405, { message = "Method not allowed" })
   end
 
   -- get parameters
   local body, err = retrieve_parameters()
   if err then
-    return responses.send_HTTP_BAD_REQUEST(err)
+    kong.log.err(err)
+    return kong.response.exit(400, { message = "Cannot process the request due to a client error" })
```

Comment:
Let's do `{message = err}` in here as this is a 400 - and no need for `kong.log.err` in this case.

Reply:
done 👍